.semaphore/heroku-production.yml

```diff
@@ -0,0 +1,25 @@
+version: v1.0
+name: Heroku deployment to production
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+blocks:
+  - name: Deploy
+    task:
+      secrets:
+        - name: book-swap-app
+      env_vars:
+        - name: HEROKU_REMOTE
+          value: https://git.heroku.com/book-swap-app.git
```

Comment:
I am now just nit-picking, but I see that the prod name for the app is `book-swap-app`, whereas for staging is `book-swap-staging`. It would be nice to have these two be consistent, so then prod would be something like `book-swap` (without the `app` part). How easy is it to change this name in Heroku?

Reply:
Yes, you are right, it would be better to have them consistent. I will change it.